src/Core/src/PublicAPI/net-ios/PublicAPI.Unshipped.txt

```diff
@@ -103,6 +103,9 @@ static Microsoft.Maui.Handlers.SearchBarHandler.MapBackground(Microsoft.Maui.Han
 static Microsoft.Maui.Handlers.SearchBarHandler.MapIsSpellCheckEnabled(Microsoft.Maui.Handlers.ISearchBarHandler! handler, Microsoft.Maui.ISearchBar! searchBar) -> void
 static Microsoft.Maui.Hosting.MauiHandlersCollectionExtensions.AddHandler<TType>(this Microsoft.Maui.Hosting.IMauiHandlersCollection! handlersCollection, System.Func<System.IServiceProvider!, Microsoft.Maui.IElementHandler!>! handlerImplementationFactory) -> Microsoft.Maui.Hosting.IMauiHandlersCollection!
 static Microsoft.Maui.Hosting.MauiHandlersCollectionExtensions.TryAddHandler<TType>(this Microsoft.Maui.Hosting.IMauiHandlersCollection! handlersCollection, System.Func<System.IServiceProvider!, Microsoft.Maui.IElementHandler!>! handlerImplementationFactory) -> Microsoft.Maui.Hosting.IMauiHandlersCollection!
+static Microsoft.Maui.Keyboard.Date.get -> Microsoft.Maui.Keyboard!
```

Comment:
The build is failing because need to include this changes in PublicAPIs also in Tizen:

```
D:\a\_work\1\s\src\Core\src\Primitives\Keyboard.cs(62,34): error RS0016: Symbol 'Date.get' is not part of the declared public API (https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md) [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net8.0-tizen]
D:\a\_work\1\s\src\Core\src\Primitives\Keyboard.cs(66,34): error RS0016: Symbol 'Time.get' is not part of the declared public API (https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md) [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net8.0-tizen]
D:\a\_work\1\s\src\Core\src\Primitives\Keyboard.cs(64,38): error RS0016: Symbol 'Password.get' is not part of the declared public API (https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md) [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net8.0-tizen]
```

Reply:
Thanks, I asked the Tizen team to implement the missing bits (: